exercises/concept/language-list/.meta/examples/languageList.example.u

```diff
@@ -0,0 +1,26 @@
+languageList.new : [Text]
+languageList.new = []
+
+languageList.add : Text -> [Text] -> [Text]
+languageList.add input list = input +: list
+
+languageList.drop1 : [Text] -> [Text]
+languageList.drop1 = cases
+  head +: tail -> tail
+  [] -> []
+
+languageList.size : [Text] -> Nat
+languageList.size list =
+  go acc = cases
+    [] -> acc
+    h :+ t -> go (Nat.increment acc) t
+  go 0 list
+
+languageList.contains : Text -> [Text] -> Boolean
+languageList.contains input = cases
+  [] -> false
+  h +: t | h === input -> true
+  h +: t -> contains input t
+
+languageList.reverse : [Text] -> [Text]
+languageList.reverse list = List.foldLeft (b a -> a List.+: b ) [] list
```

Comment:
This is using higher-order functions, which should be a prerequisite. Note that I really like that the student gets to use a function from the `List` module, as we want them understanding how to do that and where to find these functions. Another option would be to allow them to just use `List.reverse` (if there is such a function). It would then be more about finding the right function, that figuring out how to reverse lists.